### PR TITLE
feat: Add gaze.from_csv() (#504)

### PR DESCRIPTION
**Pull request by prassepaul**
## Description

Please include a clear and concise description of the change and which issue is fixed.
Please also include excepted improvements, relevant motivation and context.
List any dependencies that are required for this change.

Fixes issue #504 

## Implemented changes

Insert a description of the changes implemented in the pull request.

- [x] added from_csv
- [x] added tests

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
Provide instructions so we can reproduce.
Please also list any relevant details for your test configuration

- [x] test to load in csv file for monocular and binocular data

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by SiQube — COMMENTED** [review]
great work!

File: src/pymovements/gaze/io.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    │ 0    ┆ 0          ┆ 0          │
+    │ 0    ┆ 0          ┆ 0          │
+    │ 0    ┆ 0          ┆ 0          │
+    │ 0    ┆ 0          ┆ 0          │
+    └──────┴────────────┴────────────┘
+
+    We can now load the data into a ``GazeDataFrame`` by specyfing the experimental setting
+    and the names of the pixel position columns.
+
+    >>> gaze = from_csv(
```
[the remark this answers is not in the document]
Comment (prassepaul) [reply src/pymovements/gaze/io.py:122]: yes. will do

**Review by SiQube — APPROVED** [review]

**Comment by SiQube** [comment]
great work @prassepaul

**Review by dkrako — APPROVED** [review]
Great! Thanks a lot for your work!

**Pull request merged**